app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
Add inheritance from the `ABC` class as well for `Dwarf `, because the `Dwarf ` class does not implement all abstract methods from the `Player` class. The `Dwarf` class should be abstract as well.